modules/nf-core/platypus/meta.yml

```diff
@@ -0,0 +1,75 @@
+name: platypus
+description: Platypus is a tool that efficiently and accurately calling genetic variants from next-generation DNA sequencing data
+keywords:
+  - variant calling
+  - dna sequencing
+tools:
+  - platypus:
+      description: Platypus is a tool designed for efficient and accurate variant-detection in high-throughput sequencing data.
+      homepage: https://www.well.ox.ac.uk/research/research-groups/lunter-group/lunter-group/platypus-a-haplotype-based-variant-caller-for-next-generation-sequence-data
+      documentation: https://www.well.ox.ac.uk/research/research-groups/lunter-group/lunter-group/platypus-documentation
+      tool_dev_url: https://github.com/andyrimmer/Platypus
+      doi: "https://www.nature.com/articles/ng.3036"
+      licence: ["BSD-3-clause"]
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test']
+  - tumor_file:
+      type: file
+      description: BAM file
+      pattern: "*.bam"
+  - tummor_file_bai:
+      type: file
+      description: Index of BAMfile
+      pattern: "*.bai"
+  - control_file:
+      type: file
+      description: BAM file
+      pattern: "*.bam"
+  - control_file_bai:
+      type: file
+      description: Index of BAMfile
+      pattern: "*.bai"
```

Comment:
Can you give these a description that contains some more detail of what exactly these files are?

Reply:
applied